modules/mobile/app/services/mobile/v0/appointments/proxy.rb

```diff
@@ -59,8 +59,10 @@ def legacy_fetch_appointments(start_date, end_date)
           va_appointments = va_appointments_adapter.parse(va_response[:response].body)
           cc_appointments = cc_appointments_adapter.parse(cc_response[:response].body)
 
-          facilities = fetch_facilities(va_appointments)
-          va_appointments = backfill_appointments_with_facilities(va_appointments, facilities)
+          if va_appointments.any?
+            facilities = fetch_facilities(va_appointments)
+            va_appointments = backfill_appointments_with_facilities(va_appointments, facilities)
```

Comment:
It's still plausible that errors could occur with facilities, but I think that was a pre-existing possibility and my work didn't change it.